WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift

```diff
@@ -95,9 +95,9 @@ public class AuthenticatorAnalyticsTracker {
         /// This represents the user opening their mail. It’s not strictly speaking an in-app screen but for the user it is part of the flow.
         case emailOpened = "email_opened"
         
-        /// The screen with a username and password visible
+        /// The password challenge screen after connecting a social account to a WordPress account
```

Comment:
This isn't accurate is it? I see it being tracked in `PasswordViewController:viewDidAppear`, which means it's tracked for any access to the Password view.

Reply:
Oh, wait, nevermind. I actually thought about that, and I think you're right.

Reply:
Ha! OK, I thought about it again. Sorry! It is not currently, but will be the Password view will be displayed for email login (after Get Started). So this comment is too specific.